src/components/UserProfile.jsx

```diff
@@ -132,23 +155,41 @@ export default function UserProfile({
           /> */}
         </CardContainer>
         <CardContainer>
-          <SightingsCard
-            titleId={
-              someoneElse
-                ? 'USERS_UNPROCESSED_AGS'
-                : 'PENDING_SIGHTINGS'
-            }
-            columns={['individual', 'date', 'location', 'actions']}
-            sightings={agsData || []}
-            linkPath="pending-sightings"
-            noSightingsMsg="NO_PENDING_SIGHTINGS"
-          />
-          <SightingsCard
-            titleId={someoneElse ? 'USERS_SIGHTINGS' : 'SIGHTINGS'}
-            columns={['individual', 'date', 'location', 'actions']}
-            hideSubmitted
-            sightings={sightingsData || []}
-          />
+          {localAgsLoading && (
+            <Card titleId={'PENDING_SIGHTINGS'}>
+              <LinearProgress style={{ margin: '16px 32px' }} />
+            </Card>
+          )}
+          {!localAgsLoading && (
+            <SightingsCard
+              id="pending-sightings-card"
+              titleId={
+                someoneElse
+                  ? 'USERS_UNPROCESSED_AGS'
+                  : 'PENDING_SIGHTINGS'
+              }
+              columns={['individual', 'date', 'location', 'actions']}
+              sightings={agsData || []}
+              linkPath="pending-sightings"
+              noSightingsMsg="NO_PENDING_SIGHTINGS"
+              loading={localAgsLoading}
+            />
+          )}
+          {localSightingsLoading && (
+            <Card titleId={'SIGHTINGS'}>
+              <LinearProgress style={{ margin: '16px 32px' }} />
+            </Card>
+          )}
+          {!localSightingsLoading && (
+            <SightingsCard
+              id="sightings-card"
+              titleId={someoneElse ? 'USERS_SIGHTINGS' : 'SIGHTINGS'}
+              columns={['individual', 'date', 'location', 'actions']}
+              hideSubmitted
+              sightings={sightingsData || []}
+              loading={localSightingsLoading}
```

Comment:
this would be false always right?

Reply:
I believe you're right. I thought of just dropping the loading prop entirely from the SightingsCard components, but I figured that this would future-proof things a little better if we eventually drop the other Card element after we speed up the AGS get api call (ticket https://wildme.atlassian.net/browse/DEX-786). Happy to drop  loading={localSightingsLoading} if you agree that's the right call.